src/main/scala/codacy/stylelint/DocGenerator.scala

```diff
@@ -74,12 +77,29 @@ object DocGenerator {
     Pattern.Description(Pattern.Id(patternName), Pattern.Title(patternDescription), None, None, param)
   }
 
-  def copyDescriptionFiles(folderNames: List[String], temporaryFileLocation: String, filePathForDocs: String): Unit = {
-    val descriptionDir = File(filePathForDocs + "/description/")
+  def copyDescriptionFiles(folderNames: List[String],
+                           rulesDirectory: String,
+                           mainDirectory: File,
+                           docsDirectory: String,
+                           version: String): Unit = {
+    val descriptionDir = File(docsDirectory + "/description/")
     descriptionDir.createDirectories()
     folderNames.foreach { patternName =>
-      File(s"$temporaryFileLocation/$patternName/README.md")
-        .copyTo(File(s"$descriptionDir/$patternName.md"), overwrite = true)
+      val documentationFile = File(s"$rulesDirectory/$patternName/README.md")
+      val fileContent = documentationFile.contentAsString
+
+      // assuming local URLs start with "../" this is the pattern used at the time of this solution
+      val localUrlRegex = """\[([^]]+)]\((\.\./[^)]+)\)""".r
```

Comment:
Codacy found an issue: [Possible problematic Regex found: '"""\[([^]]+)]\((\.\./[^)]+)\)"""'](https://app.codacy.com/gh/codacy/codacy-stylelint/pullRequest?prid=6159071)

Reply:
Can you add a comment of we want to catch? Similar to the top one, I would then consider using pattern matching instead of m.group(1) and m.group(2).